plugins/_transforms.js

```diff
@@ -164,114 +164,105 @@ const mth = {
 
 /**
  * Decompose matrix into simple transforms.
- *
  * @param {TransformItem} transform
- * @param {TransformParams} params
  * @returns {TransformItem[]}
- * @see https://frederic-wang.fr/decomposition-of-2d-transform-matrices.html
+ * @see {https} ://frederic-wang.fr/decomposition-of-2d-transform-matrices.html
  */
-export const matrixToTransform = (transform, params) => {
-  const floatPrecision = params.floatPrecision;
+export const matrixToTransform = (transform) => {
+  //   const floatPrecision = params.floatPrecision;
```

Comment:
i know this PR is marked as a draft but i would expect this formatting to be fixed before it gets marked as ready